storage/innobase/xtrabackup/doc/source/advanced/log_enhancements.rst

```diff
@@ -0,0 +1,70 @@
+.. _logs:
+
+================================================================================
+Improved Log statements
+================================================================================
+
+**Percona XtraBackup** is an open-source command-line utility. Command-line tools have limited interaction with the background operations and the logs provide the progress of an operation or more information about errors.
+
+In earlier versions, the log statement did not have a standard structure. Notice in the following examples the variance in the log statements:
```

Comment:
In earlier versions, the log statements in error log or
In earlier versions, the error logs did not have..

Reply:
Updated.